app/views/dashboard/dashboard.html.erb

```diff
@@ -1,3 +1,32 @@
+<% if Time.now - current_user.created_at < 1.week %>
```

Comment:
Also, you'll probably need to add an `if current_user && ` before this to only attempt it if you're logged in. That's what's causing the failure in the tests!